lib/helpers/UiBaseLocators.ts

```diff
@@ -132,33 +134,26 @@ export class UiBaseLocators {
     await this.page.waitForTimeout(1000);
     await this.modalCaretBtn.click({force: true});
     await this.page.getByLabel(dictionaryName).click();
-    await this.chooseDictionaryBtn.click();
-  }
-
-  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
-  async addQueryBuilderWithCreateDateOption() {
-    await this.queryBuilderBtn.click({force: true});
-    await this.page.waitForTimeout(1000);
-    await this.queryBuilderOrderedBy.click({force: true});
-    await this.page.waitForTimeout(1000);
-    await this.queryBuilderCreateDate.click({force: true});
-    await this.submitBtn.click({force: true});
+    await this.chooseThreeDotsBtn.click();
   }
 
-  async addQueryBuilderWithOrderByStatement(propertyAlias: string, isAcsending: boolean) {
+  async addQueryBuilderWithOrderByStatement(propertyAlias: string, isAscending: boolean) {
     await this.queryBuilderBtn.click({force: true});
+    await this.page.waitForTimeout(500);
```

Comment:
Why do we need this hard wait here?

Reply:
The tests using this method ended up failing sometimes, as I guess the test was a bit too fast, which is why I added this wait

Reply:
You could try waitFor() instead of the hard wait.